Decisions.Equifax/CodeAndDescriptionDataConverter.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
```

Comment:
This file should be moved to a subfolder. All of our data classes are in subfolders, this will end up junking up the file structure and namespaces if we leave it. I'd suggest just creating /Shared for code that is shared between endpointw.